src/tester.rs

```diff
@@ -207,7 +208,7 @@ impl<'a> LangTester<'a> {
             for (cmd_name, mut cmd) in cmd_pairs {
                 let output = cmd
                     .output()
-                    .expect(&format!("Couldn't run command {:?}.", cmd));
+                    .unwrap_or_else(|_| panic!(format!("Couldn't run command {:?}.", cmd)));
```

Comment:
Why is this better?

Reply:
I also prefer how it was before :P

Reply:
It means that the `format` stuff isn't executed *unless* there's been a failure.

Reply:
Ah. That's beautifully non-obvious. I'd expect expect to do that.

Reply:
`expect` is a normal function, so arguments are eagerly evaluated. We can't blame `expect` here :)

Reply:
Interesting! In that case, I retract my comment.

Shame it makes the code less readable.

Reply:
I see that now yeah, that's why it takes a closure.